ui/packages/consul-ui/app/templates/dc/services/show/topology.hbs

```diff
@@ -3,7 +3,7 @@
 as |route|>
   <EventSource @src={{topology}} />
   <div class="tab-section">
-    {{#if (and (eq topology.Upstreams.length 0) (eq topology.Downstreams.length 0))}}
+    {{#if (and (eq topology.Upstreams.length 0) (eq topology.Downstreams.length 0) (not topology.DefaultAllow) (not topology.WildcardIntention))}}
```

Comment:
Does this cover for both when TProxy is enabled and when its disabled?

Reply:
Let's discuss this.